### PR TITLE
docs: reorganize the "migration from mxGraph" documentation

### PR DESCRIPTION
**Pull request by tbouffard**
Move everything from the README to the dedicated migration doc.
Clarify the existing content.

### CODE REVIEW

File: packages/docs/migrate-from-mxgraph.md [hunk 1]
```diff
@@ -3,7 +3,34 @@
 **This page is a work in progress. Comments are welcome by creating an [issue](https://github.com/maxGraph/maxGraph/issues)
 or opening a [discussion](https://github.com/maxGraph/maxGraph/discussions/categories/q-a).!**
 
-The `maxGraph` API is close to the former `mxGraph` API. This page aims to help the migration to the new API.  
+The `maxGraph` APIs are not fully compatible with `mxGraph` APIs, but the `maxGraph` APIs are close to the former `mxGraph` APIs.
```
Comment (csouchet) [inline packages/docs/migrate-from-mxgraph.md]:
```suggestion
The `maxGraph` APIs are not fully compatible with the `mxGraph` APIs, but the `maxGraph` APIs are close to the former `mxGraph` APIs.
```

File: packages/docs/migrate-from-mxgraph.md [hunk 2]
```diff
@@ -3,7 +3,34 @@
...
+The concepts are the same, so experienced _mxGraph_ users should be able to switch from _mxGraph_ to _maxGraph_ without issues.
+
+The major changes are the support removal of Internet Explorer (including VML support) and Legacy Edge.
```
Comment (csouchet) [inline packages/docs/migrate-from-mxgraph.md]:
```suggestion
The major changes are the removal of support for Internet Explorer (including VML support) and Legacy Edge.
```

File: packages/docs/migrate-from-mxgraph.md [hunk 3]
```diff
@@ -3,7 +3,34 @@
...
+
+
+## Application setup
+
+Remove the
+- `mxgraph` dependency and add `maxgraph@core` instead
```
Comment (csouchet) [inline packages/docs/migrate-from-mxgraph.md]:
```suggestion
- `mxgraph` dependency and add `maxgraph@core` instead.  
```

File: packages/docs/migrate-from-mxgraph.md [hunk 4]
```diff
@@ -3,7 +3,34 @@
...
+The concepts are the same, so experienced _mxGraph_ users should be able to switch from _mxGraph_ to _maxGraph_ without issues.
+
+The major changes are the support removal of Internet Explorer (including VML support) and Legacy Edge.
+
+
+## Application setup
+
+Remove the
+- `mxgraph` dependency and add `maxgraph@core` instead  
+- _mxGraph_ initialization code using the `factory` function. _maxGraph_ objects can be accessed directly.
```
Comment (csouchet) [inline packages/docs/migrate-from-mxgraph.md]:
```suggestion
- _mxGraph_ initialization code using the `factory` function. Access _maxGraph_ objects directly.
```

File: packages/docs/migrate-from-mxgraph.md [hunk 5]
```diff
@@ -3,7 +3,34 @@
...
+- `@typed-mxgraph/typed-mxgraph` dependency in the `package.json` file
+- `typeroots` settings related to `typed-mxgraph` in the `tsconfig.json` file
+
+As an example, you can check [this project](https://github.com/maxGraph/maxgraph-integration-examples/tree/main/projects/rollup-ts) which has been migrated
+from a [typed-mxgraph example](https://github.com/typed-mxgraph/typed-mxgraph-example-bundled-with-rollup).
+
+
+## General guidelines
+
+- The names of _mxGraph_ objects were all prefixed by `mx`. The prefix has been dropped in _maxGraph_.
```
Comment (csouchet) [inline packages/docs/migrate-from-mxgraph.md]:
```suggestion
- The names of _mxGraph_ objects were all prefixed with `mx`. The prefix has been dropped in _maxGraph_.
```

File: packages/docs/migrate-from-mxgraph.md [hunk 6]
```diff
@@ -3,7 +3,34 @@
...
+- Most names remain the same
```
Comment (csouchet) [inline packages/docs/migrate-from-mxgraph.md]:
```suggestion
- Most names remain the same.
```

**Review by csouchet — COMMENTED** [review]
ℹ️ Some suggestion from Deepl.
You are free to commit them or not 😉

**Comment by tbouffard** [comment]
@csouchet I have just updated the documentation with your suggestions.

**Review by csouchet — APPROVED** [review]

**Pull request merged**